examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -97,19 +95,42 @@
 # with different estimators
 estimators = [
     BayesianRidge(),
-    DecisionTreeRegressor(max_features="sqrt", random_state=0),
-    ExtraTreesRegressor(n_estimators=10, random_state=0),
+    RandomForestRegressor(
+        # We tuned the hyperparameters of the ExtraTreesRegressor to minimize
+        # the execution time
+        n_estimators=4,
+        max_depth=10,
+        bootstrap=True,
+        max_samples=0.5,
+        n_jobs=2,
+        random_state=0,
+    ),
+    ExtraTreesRegressor(
+        # We tuned the hyperparameters of the ExtraTreesRegressor to minimize
+        # the execution time
+        n_estimators=4,
+        max_depth=10,
+        bootstrap=True,
+        max_samples=0.5,
+        n_jobs=2,
+        random_state=0,
+    ),
     KNeighborsRegressor(n_neighbors=15),
 ]
 score_iterative_imputer = pd.DataFrame()
-for impute_estimator in estimators:
+tolerances = (1e-3, 1e-1, 1e-1, 1e-2)
+# iterative imputer is sensible to the tolerance and
```

Comment:
I would move this comment above the `tolerances` variable

Reply:
and maybe emphasize that we tuned the tolerance to keep this example run with limited computational resources while not changing the results too much compared to keeping the stricter default value for the tolerance parameter.